src/usr/lib/rsetup/cli/u-boot-menu.sh

```diff
@@ -61,7 +61,7 @@ disable_overlays() {
 __rebuild_overlays_worker() {
     local overlay="$1" new_overlays="$2"
 
-    if [ -n "$(dtbo_is_compatible "$overlay")" ]
+    if [[ -n $(dtbo_is_compatible "$overlay") ]]
```

Comment:
```suggestion
    if [[ -n “$(dtbo_is_compatible "$overlay")” ]]
```
虽然bash不会把$()输出的带空格内容解释为语法错误，但由于其他的地方很多时候仍需要将其用引号包裹，从代码规范的角度是要求始终用引号，以免在进行代码复制粘贴的时候粘贴到了没有语法保护的地方